src/main/java/com/googleinterns/zoomtube/transcriptParser/TranscriptParser.java

```diff
@@ -33,7 +36,10 @@
 
 /** Parses the transcript XML and stores the lines in datastore. */
 public final class TranscriptParser {
-  private static final String XML_URL_TEMPLATE = "http://video.google.com/timedtext?lang=en&v=";
+  private static final String API_URL = "http://video.google.com/timedtext";
```

Comment:
What is this video? Can you add a video?

Reply:
This isn't a video, this is the `timedtext` API, which we use for our transcript.

Reply:
All your variables are have API_, but it isn't mentioned what the API is maybe add a comment?

Reply:
Added the API name to the class javadoc

Reply:
That is good, not everyone reads the class-wish JavaDoc so maybe just add a comment above API_URL?